frontend/src/component/filter/FilterItem/FilterItemChip/FilterItemChip.tsx

```diff
@@ -31,6 +31,7 @@ const StyledLabel = styled('div')(({ theme }) => ({
     alignItems: 'center',
     justifyContent: 'space-between',
     fontWeight: theme.typography.fontWeightBold,
+    minHeight: theme.spacing(3.5),
```

Comment:
we need to make sure that filters with chip and without a chip have the same height